isis/src/base/apps/pds2isis/pds2isis.xml

```diff
@@ -44,18 +44,13 @@ xsi:noNamespaceSchemaLocation=
       example, if NULLMIN=0.0, NULLMAX=3.0, LISMIN=3.0, and LISMAX=5.0, the actual
       raw value 3.0 can be assigned as a NULL or LIS, but is translated to NULL because
       NULL has a higher priority than LIS.
-    <p>
-      It should be noted for 8-bit images
-      that ISIS uses zero as it's default NULL value so any
-      <def link="Digital Number">pixel value</def> of zero will be convert to a
-      NULL value in the output cube. It is recommended that any 8-bit images
-      either be converted to 16-bit images or run the following:
     </p>
     <p>
-      <b>pds2isis from=input8bit.img to=output8bit.cub</b>
-      <br/>
-      <b>stretch from=output8bit.cub to=output16bit.cub+16bit NULL=0</b>
-    </p>
+      It should be noted if any incoming <def link="Digital Number">pixel values</def>
+      are equal to one of the ISIS <def link="Special Pixel">special pixel</def>
+      values for the corresponding bit type, those pixels will be special in the
+      output cube. If you are working with an 8-bit image and need either 0
+      or 255 either convert the data to 16-bit or use Stretch after ingestion.
```

Comment:
Missing ","
Think we should not single out 8 bit here. Either explain the details of all bit types here or maybe explain in general terms of special pixels and make sure the links point to the docs that show what the values are for specials.

Reply:
It's probably worth mentioning 8-bit in particular here because it's where this is most likely to come up. Some blurb saying that all DNs that are special pixel values for the bit type will not be interpreted by ISIS as valid pixels, here's the special pixel values for each bit type, this is most commonly an issue for 8-bit images, and then the stretch work-around.

Reply:
@scsides Where exactly is missing a ","? I just want to add it to the appropriate location first go around.

Reply:
between "255 either"